discord/bot.go

```diff
@@ -662,13 +662,347 @@ func (guild *GuildState) handleMessageCreate(s *discordgo.Session, m *discordgo.
 						guild.GameStateMsg.AddReaction(s, e.FormatForReaction())
 					}
 					guild.GameStateMsg.AddReaction(s, "❌")
+				case Settings:
```

Comment:
I really really like the functionality you added; you actually beat me to it, I was prototyping a version on my own branch!
However, I really can't merge changes that add another 300 lines to a file I was already struggling with, length-wise haha. I think it would be pretty easy to move to another file though, maybe settings.go?

Reply:
Sure, I'll work on it and implement the other suggestions.